### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/3454

@JohanFreelancer9 do you think I should change _among_ with _by_ as the issue says? Thanks

### CODE REVIEW

File: SharePoint/SharePointOnline/information-architecture-models-examples.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ Common methods of organizing navigational design:
 
 **Benefits** - Organizing by geographic location is an effective way to bring people of similar disciplines together to collaborate and build community. For large organizations, and international organizations, location can be an important decision-making factor for users. Organizing by region may be important when certain topics have different laws and mandates depending on the region. For example, crisis management sites or human resource guidelines. 
 
-**Governance considerations** - Many governance details may very among region, for example site creation policy, data retention, and data storage policies too. Learn more about the [multi-geo user experience](/microsoft-365/enterprise/multi-geo-user-experience) and [multi-geo configuration](/sharepoint/dev/solution-guidance/multigeo-introduction).
+**Governance considerations** - Many governance details may vary among region, for example site creation policy, data retention, and data storage policies too. Learn more about the [multi-geo user experience](/microsoft-365/enterprise/multi-geo-user-experience) and [multi-geo configuration](/sharepoint/dev/solution-guidance/multigeo-introduction).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/information-architecture-models-examples.md:67]:
```suggestion
**Governance considerations** - Many governance details may vary by region, for example site creation policies, data retention, and data storage policies. Learn more about the [multi-geo user experience](/microsoft-365/enterprise/multi-geo-user-experience) and [multi-geo configuration](/sharepoint/dev/solution-guidance/multigeo-introduction).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Hi @get-itips, yes "by region" is correct in this instance. Also, two other changes that the submitter might have overlooked. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MattEEvans PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by MattEEvans — APPROVED** [review]
Changes look good. Thanks!

**Pull request merged**